src/meltano/cli/params.py

```diff
@@ -29,6 +37,16 @@ def decorate(*args, database_uri=None, **kwargs):
     return functools.update_wrapper(decorate, func)
 
 
+install_option = click.option(
+    "--install/--no-install",
+    "install_plugins",
+    default=True,
```

Comment:
Should this default to `False` to make it opt-in?

```suggestion
    default=False,
```

Reply:
Depends on what you think is best. I know you mentioned opt-in earlier and I did mean to point out before that it is currently opt-out, but I implemented it this way initially because it would mean users immediately get the benefit of auto-install. Given that a plugin is already installed, there shouldn't be any new behaviour here running any of the affected commands with respect to that plugin. Maybe I'm missing something (maybe CI or Docker use-cases)?

Also, since `meltano add` shares this `--install/--no-install` implementation now, the default has to be `true` to keep the current behaviour (although this should be fairly straightforward to make an exception for if we do change the default).

Reply:
Gotcha.

I'm looking through the code now and it's looking great! Each invocation of `install_plugins()` just gets called with the relevant plugins, so that eases my concern for projects with a ton of plugins 🙌.

I may just have a thought about the function signatures.